Comp2023/src/main/java/frc/robot/commands/AutoChargeStation.java

```diff
@@ -0,0 +1,48 @@
+
+// ROBOTBUILDER TYPE: SequentialCommandGroup.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.Swerve;
+
+/**
+ *
+ */
+public class AutoChargeStation extends SequentialCommandGroup
+{
+  public AutoChargeStation(Swerve swerve)
+  {
+    setName("AutoChargeStation");
+
+    addCommands(
+        // Add Commands here:
+
+        // @formatter:off
+        new PrintCommand("AUTO PATH SEQUENCE: go to ChargeStation"),
+        new ParallelDeadlineGroup(
+          new WaitUntilCommand(swerve::driveWithPathFollowerIsFinished),
+          new AutoDrivePath (swerve, "driveoffcommunity", true)

```

Comment:
I think "driveOffCommunity" will run a 2-3 meter path that is different than just driving up onto the charge station, no?

Reply:
Yes, I think the distances are slightly different